src/core_plugins/kibana/public/home/home.less

```diff
@@ -26,3 +26,81 @@ home-app {
 .sampleDataSetCard {
   flex-grow: 0;
 }
+
+.home-welcome {
```

Comment:
@chrisdavies there is a subtle background gradient that needs added here. It's not super apparent in the marvel link, but if you select the background layer, then you'll see this:

```background-image: linear-gradient(0deg, #F5F5F5 0%, #FFFFFF 100%);```

If you have access to the EUI color codes, then you could use *$euiColorLightestShade* and *$euiColorEmptyShade* , respectively.

Reply:
Noice! I'll do it.